dev/devicelab/bin/tasks/route_test_ios.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
do you also have to update the .ci.yaml?

Reply:
Yes but only for the Mac-IOS; the Linux android stays the same

Reply:
Oops, I didn't see that this PR had a diff on .ci.yaml. It was a long day yesterday :P